Jellyfin/Jellyfin.csproj

```diff
@@ -2,6 +2,7 @@
 <Project ToolsVersion="15.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
   <PropertyGroup>
+    <LangVersion>8.0</LangVersion>
```

Comment:
C# 8 was released in September 2019. You should upgrade to Visual Studio 2022 as I suspect you're using 2019.

Reply:
I believe UWP only supports up to 8. I am using the newest Visual Studio version 2022 17.12. https://learn.microsoft.com/en-us/answers/questions/1462666/how-to-change-c-version-of-a-uwp-program